sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/degrade/DegradeRule.java

```diff
@@ -55,7 +55,7 @@
  */
 public class DegradeRule extends AbstractRule {
 
-    private static final int RT_MAX_EXCEED_N = 5;
+    private int rtMaxExceedN = RuleConstant.DEGRADE_GRADE_RT_MAX_EXCEED_N;
```

Comment:
The `RT_MAX_EXCEED_N` has two meanings actually:

- For RT mode, it indicates the minimum number of consecutive slow requests that can trigger RT circuit breaking.
- For exception ratio mode, it indicates the minimum number of requests (in an active statistic time span) that can trigger circuit breaking.

So maybe we need two attributes here? And the attribute name might need to be discussed.

Reply:
hi, @sczyh30 
```
        DegradeRule rule = new DegradeRule();
        rule.setResource(KEY);
        // set limit exception ratio to 0.1
        rule.setCount(0.1);
        rule.setGrade(RuleConstant.DEGRADE_GRADE_EXCEPTION_RATIO);
        rule.setTimeWindow(10);
        rule.setRtMaxExceedN(20);
        rules.add(rule);
```
The downgrade rule can only be set to one mode at present. rt mode and exception ratio mode Only one can be selected. The RT_MAX_EXCEED_N attribute can represent different meanings in different modes. So I think that using only one attribute is ok. and you ?

Reply:
They're indeed two different things, so only one attribute might be confusing. I think we can make it two attributes:

- `rtSlowRequestAmount` for minimum number of consecutive slow requests that can trigger RT circuit breaking.
- `minRequestAmount` for the minimum number of requests (in an active statistic time span) that can trigger circuit breaking.

What do you think of it? @CarpenterLee

Reply:
Any progress?

Reply:
two attribute may be better